### PR TITLE
[FSTORE-577] update spark options for s3

### PR DESCRIPTION
**Pull request by DhananjayMukhedkar**

### CODE REVIEW

File: docs/user_guides/fs/storage_connector/creation/s3.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ With this option the [encryption key is managed by AWS KMS](https://docs.aws.ama
 
 If you have SSE-KMS enabled for your bucket, you can find the key ARN in the "Properties" section of the bucket details on AWS.
 
+### Step 5: Add Spark Options (optional)
+Here you can specify any additional spark options that you wish to add to the spark context at runtime. Multiple options can be added as key - value pairs.
+
+!!! tip
+To connect to a non-AWS hosted S3 compliant storage you can add the option with key as `fs.s3a.endpoint` and the endpoint you want to use as value. The storage connector will then be able to read from your specified S3 compatible storage.
```
Comment (SirOibaf) [inline docs/user_guides/fs/storage_connector/creation/s3.md:75]: You need to indent this paragraph for the layout to work correctly.

Comment (SirOibaf) [inline docs/user_guides/fs/storage_connector/creation/s3.md:75]: `To connect to a S3 compatible storage, which is not AWS S3, you can add ...`

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: docs/user_guides/fs/storage_connector/creation/s3.md (hunk 1 above)
Comment (DhananjayMukhedkar) [reply docs/user_guides/fs/storage_connector/creation/s3.md:75]: done

Comment (DhananjayMukhedkar) [reply docs/user_guides/fs/storage_connector/creation/s3.md:75]: done

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**